components/spinner/__docs__/storybook-stories.jsx

```diff
@@ -18,6 +18,16 @@ const inverseContainer = (getStory) => (
 	<div style={inverseContainerStyle}>{getStory()}</div>
 );
 
+const inlineContainerStyle = {
```

Comment:
The [Inlined blueprint](https://www.lightningdesignsystem.com/components/spinners/#Inlined) uses a container with: `<div class="slds-align_absolute-center" style="height:4rem">`.  I think we should stick with the same example here.

I think that might be related why the spinner in Storybook looks weirdly off screen for me:

<img width="365" alt="image" src="https://user-images.githubusercontent.com/7608505/87358741-445dac80-c51b-11ea-8395-dc4e0788e6fd.png">

After removing the `<div class="slds-spinner_container">` element and switching the container to use `<div class="slds-align_absolute-center" style="height:4rem">`, it looks a little better:

<img width="1075" alt="image" src="https://user-images.githubusercontent.com/7608505/87358869-8a1a7500-c51b-11ea-8537-acaf42eb8923.png">